src/main/java/com/benko/exercise/sort/BubbleSorter.java

```diff
@@ -0,0 +1,34 @@
+package com.benko.exercise.sort;
+
+public class BubbleSorter {
+
+    public void sort(int[] numbers) {
+        for (int i = 0; i < numbers.length; i++) {
+            swapElements(numbers);
+        }
+        printNumbers(numbers);
+    }
+
+    private static int[] swapElements(int[] numbers) {
+        for (int i = 0; i < numbers.length-1; i++) {
+            if (isGeraterThan(i, i+1, numbers)) {
+                int temp;
+                temp = numbers[i + 1];
+                numbers[i + 1] = numbers[i];
+                numbers[i] = temp;
+            }
+        }
+        int[] swapedElements = numbers;
+        return swapedElements;
+    }
+
+    private static boolean isGeraterThan(int x, int y, int[] numbers) {
```

Comment:
gerater?

Reply:
jak jest crated to może być gerater :P
corrected